src/app/views/pages/home/home.component.html

```diff
@@ -290,7 +290,7 @@
                               <div class="ms-30p">
                                  <img class="semi-colun" src="../../../../assets/home-asset/semi-colun.png" alt="semi-colun">
                               </div>
-                              <div class="text-blue fw-900 fs-24 mt-5">{{ "HOME_M_R_MAHESWARI" | i18next }}</div>
+                              <div class="text-blue fw-900 fs-24 mt-5">{{ "HOME_M_R_Maheswari" | i18next }}</div>
```

Comment:
is this required? either revert it or add the same const in tamil json

Reply:
@gouravmore it's not required now but may be require in future

Reply:
Can you switch to the Tamil lang and check again

Reply:
yes checked @gouravmore it's a event when we clicked on let's play it will redirect us to "Boost your learning with Super language Lab" section